webapp/packages/plugin-data-viewer/src/DatabaseDataModel/Actions/ResultSet/isResultSetBinaryFileValue.ts

```diff
@@ -1,5 +1,13 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
 import type { IResultSetBinaryFileValue } from './IResultSetBinaryFileValue';
+import { isResultSetContentValue } from './isResultSetContentValue';
 
 export function isResultSetBinaryFileValue(value: any): value is IResultSetBinaryFileValue {
-  return value?.contentType === 'application/octet-stream' && Boolean(value?.binary);
+  return isResultSetContentValue(value) && value?.contentType === 'application/octet-stream' && 'binary' in value;
 }
```

Comment:
please rename to `isResutSetBinaryValue` to prevent meaning conflict with `isResultSetFileValue`

Reply:
also it seems like we can use this helper in:
[if (isResultSetContentValue(this.cellValue) && this.cellValue.binary) {](https://github.com/dbeaver/cloudbeaver/blob/d1d2d66fb6c94b0bd4b6dd89f31e2cdc53f86342/webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/ImageValue/ImageValuePresentation.tsx#L125-L126)

and 
[if (isResultSetContentValue(this.cellValue)) {](https://github.com/dbeaver/cloudbeaver/blob/d1d2d66fb6c94b0bd4b6dd89f31e2cdc53f86342/webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/ImageValue/ImageValuePresentation.tsx#L156-L157)

Reply:
and rename `IResultSetBinaryFileValue` in the same way